client/src/components/admin/teams/TeamsView.jsx

```diff
@@ -75,15 +81,28 @@ class TeamsView extends Component {
         e.isChecked = false;
       }
     });
-    this.setState({ cohorts: newCohortList });
+    const selectedCohort = newCohortList.filter(c => c.isChecked);
```

Comment:
Please only use single character variables where convention has already established their meaning.  (i, j for loops, e for errors or events).

change `const selectedCohort = newCohortList.filter(c => c.isChecked);` to 
`const selectedCohort = newCohortList.filter(cohort => cohort.isChecked);`